### ISSUE TITLE
No accessible text for nav links

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4067]
## Area or Component
Navigation bar in the main app

## Quick Summary
The links for "my account" and "settings" don't have discernable text (it's just the URL of the button, so `/admin/facility?id=xyz` or similar)

### How to Replicate
Inspect the elements in the header and see that there's no accessible text for their purpose

## Accessibility Violation
Links must have discernible text: https://dequeuniversity.com/rules/axe/4.4/link-name?application=AxeChrome

## Action Requested
Add accessible text for these buttons (`aria-label` would probably do it)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=981460209&range=24:24

**Comment by emmastephenson** [comment]
@BobanL, I think your PR #4190 will also address this, is that correct?

**Comment by BobanL** [comment]
Yes thats right!
